gateway/src/cluster/event.rs

```diff
@@ -0,0 +1,93 @@
+//! Events that the cluster emits to event streams.
+//!
+//! Included is the larger [`Event`] exposed to event streams. It contains
+//! variants with all of the possible events that can come in: new channels,
+//! heartbeat acknowledgements, "meta" events of when a shard disconnects or
+//! connects, etc.
+//!
+//! Also included is the [`EventType`] bitflags, which can be used to identify
+//! the type of an event and to filter events from event streams via
+//! [`ClusterBuilder::event_types`].
+//!
+//! [`EventType`]: ::twilight_model::gateway::event::EventType
```

Comment:
nit: don't think the `::` is needed?

Reply:
Resolved. That was copied from `shard::Events`; I'll do a separate PR to remove them.